README.md

```diff
@@ -84,17 +86,17 @@ that describes it for both Unix and Windows systems.
 
 ## Signed Releases
 
-As of v0.10.15 and onwards release binaries are GPG signed with the key
-D26E6ED000654A3E, available from https://syncthing.net/security.html and
-most key servers.
+Beginning with `v0.10.15`, release binaries are GPG-signed with the key
+`D26E6ED000654A3E`, available from https://syncthing.net/security.html and
+most public key servers.
```

Comment:
I think Markdown syntax specifies backticks (` `` `) as code markers.  Not a mixture of backtick and single-quote.